### ISSUE TITLE
action invoke requires leading slash on packaged action

### ISSUE THREAD

**Issue by rabbah** [nimbella/aio-cli-plugin-runtime#27]
```
> nim action invoke whisk-system/utils/context   
 ›   Error: failed to invoke the action: the requested resource does not exist. (404 Not Found)
```

but this works:
```
> nim action invoke /whisk-system/utils/context  
{...}
```

**Comment by rabbah** [comment]
this one is ugly - the real issue is in npm openwhisk, which tolerates an optional leading `/` when it parses out the namespace and asset name from the `options.name` property but then drops this information when it checks explicitly if the given name starts with a `/` and proceeds to ignore the parsed namespace.

**Comment by rabbah** [comment]
Making the change in npm openwhisk is difficult in my assessment. Instead, I've taken as a requirement that the name should be fully qualified when using the SDK. So accordingly, qualify the names fully on the client.

**Comment by rabbah** [comment]
Closed by 0e1f6c6ecc768b1ef1c6362651a85032511ec22d - will upstream to aio.
